packages/@react-types/form/src/index.d.ts

```diff
@@ -57,8 +57,8 @@ export interface FormProps extends AriaLabelingProps {
    */
   autoComplete?: 'off' | 'on',
   /**
-   * A nonstandard attribute used by iOS Safari that controls how textual form elements should be automatically capitalized.
-   * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#autocapitalize).
+   * Controls whether inputted text is automatically capitalized and, if so, in what manner. 
+   * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize).
```

Comment:
I have updated the comment and link to reflect changes in MDN's [description of autocapitalize](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize).

ref: 
- https://github.com/mdn/content/pull/30775
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize